pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -95,3 +95,84 @@ mod organizer {
 		});
 	}
 }
+
+#[cfg(test)]
+mod season_metadata {
+	use super::*;
+
+	const ALICE: u32 = 1;
+	const BOB: u32 = 2;
+
+	const METADATA: SeasonMetadata = SeasonMetadata { name: [0; 100], description: [0; 1000] };
+
+	const SEASON_ID: SeasonId = 0;
+
+	#[test]
+	fn update_season_metadata_happy_path() {
```

Comment:
Let's go with `function_should_do_something` naming conventions
```suggestion
	fn update_season_metadata_should_work() {
```